### PR TITLE
Python3 fixes to condb scripts

### PR DESCRIPTION
**Pull request by davidlange6**
Fixes to python scripts not ending in .py. Applied some futurize and autopep8 fixers and added @mrodozov changes as well (which I didn't yet test on python2 but look safe)

#### PR validation:

failure rate of python3 unit tests reduces

### CODE REVIEW

**Comment by davidlange6** [comment]
please test

File: CondCore/Utilities/python/conddblib.py [hunk 1]
```diff
@@ -485,7 +485,7 @@ def _getCMSFrontierConnectionString(database):
 def _getCMSSQLAlchemyConnectionString(technology,service,schema_name):
     if technology == 'frontier':
         import urllib
-        return '%s://@%s/%s' % ('oracle+frontier', urllib.quote_plus(_getCMSFrontierConnectionString(service)), schema_name )
+        return '%s://@%s/%s' % ('oracle+frontier', urllib.parse.quote_plus(_getCMSFrontierConnectionString(service)), schema_name )
```
Comment (davidlange6) [inline CondCore/Utilities/python/conddblib.py]: hi @mrodozov - i gather that this change is not python2 compatible.

**Comment by davidlange6** [comment]
please test

**Comment by davidlange6** [comment]
please test

**Comment by davidlange6** [comment]
seems to be an unrelated db connection issue?

**Comment by davidlange6** [comment]
please test

**Comment by mrodozov** [comment]
test_MpsWorkFlow is exactly what fails. not sure about the relvals

**Comment by fabiocos** [comment]
please test

**Comment by davidlange6** [comment]
Yea the relvals failures were glitches


David


On 14 Oct 2019, at 14:12, Mircho Rodozov <notifications@github.com> wrote:

﻿

test_MpsWorkFlow is exactly what fails. not sure about the relvals

**Comment by fabiocos** [comment]
@davidlange6 humm, that should have been fixed already by #28150 by @smuzaffar , but apparently there is another issue, in any case it does not look related to this PR.

**Comment by davidlange6** [comment]
ah - no wonder I couldn't find the bug with this PR last night... maybe we can get this for the 1100 IB?

**Comment by fabiocos** [comment]
@davidlange6 ok, I am just not sure about the difference added by @mrodozov wrt your version, discussed in https://github.com/cms-sw/cmssw/pull/28161#issuecomment-541789498 
I can anyway take care of rebasing his own version

**Comment by fabiocos** [comment]
+1

**Comment by fabiocos** [comment]
@ggovi all this should be technical as discussed yesterday, please have a look anyway

**Comment by fabiocos** [comment]
merge

**Comment by ggovi** [comment]
+1

**Comment by smuzaffar** [comment]
@fabiocos , we need to revert this. This is breaking many unit tests

**Comment by davidlange6** [comment]
looks to be all one error (and in a few packages) - lets see about fixing before 1100..

**Pull request merged**